src/options.cpp

```diff
@@ -135,7 +135,14 @@ void options_manager::addOptionToPage( const std::string &name, const std::strin
 {
     for( Page &p : pages_ ) {
         if( p.id_ == page ) {
+            // Don't add duplicate options to the page
+            for( const cata::optional<std::string> &i : p.items_ ) {
+                if( i.has_value() && i.value() == name ) {
```

Comment:
Shouldn't duplicates be handled in the calling code? I mean something tries to add duplicate entries and that's weird.

Reply:
Well, yes, `DISPLAY` option is added both through `find_videodisplays` and `add_graphic_options`.